src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterSuggestTests.cs

```diff
@@ -192,8 +193,11 @@ public void TestSuggestMutationFunctions(string expression, params string[] expe
 
         [Theory]
         [InlineData("Collect(|", "Entity1", "Entity2", "Table1", "table2")]
+        [InlineData("Collect(e|", "Entity1", "Entity2", "Filter", "Table1", "table2", "false", "Self", "true")]
         [InlineData("Patch(|", "Entity1", "Entity2", "Table1", "table2")]
+        [InlineData("Patch(t|", "Distinct", "Entity1", "Entity2", "Filter", "Sort", "Table1", "table2", "Not", "true")]
```

Comment:
>"Filter", "Sort" [](http://example.com/codeflow?start=66&length=16)

The way we will still defer from CallNode is, In this we also suggest function names (and we ought to for correctness, but with this PR the list is shrunk to only function whose return type matches with the argument type.

Maybe we should also consider suggesting function in call node? (Ie the test one line above)